### ISSUE TITLE
Handle gz'ed images: Unexpected image format ... rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz

### ISSUE THREAD

**Issue by yprokule** [openshift/ironic-rhcos-downloader#11]
*coreos-downloader* image fails:
``` bash
sudo podman ps -a
CONTAINER ID  IMAGE                                                         COMMAND               CREATED         STATUS                     PORTS                   NAMES
746ec1901210  registry.svc.ci.openshift.org/ocp/4.3-2019-11-12-053448:none  /usr/local/bin/ge...  26 minutes ago  Exited (1) 5 minutes ago                           coreos-downloader
``` 
And log shows it fails due to **Unexpected image format**:
```bash
sudo podman logs 746ec1901210
+ export RHCOS_IMAGE_URL=https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz
+ RHCOS_IMAGE_URL=https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz
+ '[' -z https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz ']'
++ echo https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz
++ cut -f 1 -d '?'
+ RHCOS_IMAGE_URL_STRIPPED=https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz
+ [[ https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz = *.qcow2 ]]
+ echo 'Unexpected image format https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz'
Unexpected image format https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz
+ exit 1
+ export RHCOS_IMAGE_URL=https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz
+ RHCOS_IMAGE_URL=https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz
+ '[' -z https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz ']'
++ echo https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz
++ cut -f 1 -d '?'
+ RHCOS_IMAGE_URL_STRIPPED=https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz
+ [[ https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz = *.qcow2 ]]
+ echo 'Unexpected image format https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz'
Unexpected image format https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911081536.0/x86_64/rhcos-43.81.201911081536.0-openstack.x86_64.qcow2.gz
+ exit 1
```

**Comment by yprokule** [comment]
/assign @stbenjam 

is it fixed by https://github.com/openshift/ironic-rhcos-downloader/pull/12 ?

**Comment by hardys** [comment]
Fixed via #12 #13 and #14
